ui/packages/admin-security-certificate-ui/src/main/webapp/templates/certificateDetail.handlebars

```diff
@@ -18,7 +18,7 @@
 {{else}}
 {{#each value}}
  <div class="cert-callout">
-     <h4 class="cert-title">Certificate {{@index}}</h4>
+     <h4 class="cert-title">Certificate {{inc @index}}</h4>
```

Comment:
An alternative to this would be to update the serializeData method on the view using this template to feed in the value you need, but this works just as well.

Reply:
👍